src/commands/deploy.js

```diff
@@ -90,22 +92,21 @@ const createEmbulkConfig = (mashrConfigObj) => {
   return yaml.safeDump(embulkConfig);
 }
 
-const createCrontab = (runCommand) => {
+const createEmbulkScript = (runCommand) => {
   // TODO: place logs in stackdriver
   // TODO: what to do with logs? Does the log file get too large?
   // diff file run from root of container. Can't use it after?
-  let crontabCommand = runCommand;
-  crontabCommand = crontabCommand.replace(
-  'embulk_config.yml', '/app/mashr/embulk_config.yml.liquid');
-
-  const crontab =
-`PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
-DATE=date +"%Y-%m-%d-%H_%M_%S_%3N;
-export DATE
-* * * * * ${crontabCommand} >> /var/log/cron.log 2>&1
-# An empty line is required at the end of this file for a valid cron file.
+  runCommand = runCommand.replace(
+    'embulk_config.yml', '/root/mashr/embulk_config.yml.liquid');
```

Comment:
I'm getting confused about what the directory structure is on the VM for mashr. Is it:

`/root/mashr/app`
Or
`/root/mashr`
Or
`/app/mashr/...`
?

Reply:
The thing that helps me is to try and distinguish the directory structure for the VM and then for the container. It can be confusing because they're slightly different. It helps to look at the "startup script" that we pass into the function that creates the GCE instance, and the Dockerfile we use to build our container image. It also helps to run the deploy script and walk through both the VM and container instance. Stepping through it might help the most.